.ado/publish.yml

```diff
@@ -124,7 +124,7 @@ jobs:
     displayName: Install Prereqs and set version
 
   - script: |
-      python ./build.py --jupyterlab --widgets --no-check
+      python ./build.py --npm --jupyterlab --widgets --no-check
```

Comment:
Ugh, just realized npm depends on wasm, so we'll end up building the whole thing here. Will think about it, but might go back to ignoring the import warning from TypeScript when importing 'out of tree' modules :-/

Reply:
Fixed it via a tsconfig.json change, so no build changes needed - thankfully.